src/server/connection.cpp

```diff
@@ -27,14 +27,14 @@ http::compression_type select_compression(const boost::beast::http::fields &fiel
 {
     const auto header_value = fields[boost::beast::http::field::accept_encoding];
     /* giving gzip precedence over deflate */
-    if (boost::icontains(header_value, "deflate"))
-    {
-        return http::deflate_rfc1951;
-    }
     if (boost::icontains(header_value, "gzip"))
```

Comment:
Well, right now the server's compression preference is the one that works :slightly_smiling_face: 
But yes, we should improve this. Another reason why it would be good to replace the custom HTTP server in `osrm-routed`

Reply:
I didn't mean to do it now, it was just a note :)